<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCurveEditor</class>
 <widget class="QWidget" name="MCurveEditor">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCurveEditorMainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="MCurveEditorMainLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QFrame" name="MCurveEditorCurveArea">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="MCurveEditorScrollWidgetContentsLayout">
          <item row="3" column="0">
           <widget class="QScrollBar" name="SvcEditorHorizontalScrollBar">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="MCurveEditorPage" name="VolumeWidget" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="MCurveEditorPage" name="SpkMixWidget" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="MCurveEditorPage" name="F0Widget" native="true"/>
          </item>
          <item row="2" column="1">
           <widget class="QScrollBar" name="VolumeScrollBar">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QScrollBar" name="F0ScrollBar">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="pageStep">
             <number>99999999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QScrollBar" name="SpkScrollBar">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="invertedAppearance">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" rowspan="2">
      <widget class="QFrame" name="MCurveEditorRight">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>125</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="MMenuLayout">
          <item row="10" column="0" colspan="2">
           <widget class="QPushButton" name="SaveSpeakerMixToNpy">
            <property name="text">
             <string>SaveSpeakerMixToNpy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <spacer name="SvcEditorSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QPushButton" name="SvcEditorOpenAudioButton">
            <property name="text">
             <string>SvcEditorOpenAudioButton</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="SamplingRateLabel">
            <property name="text">
             <string notr="true">SR:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QComboBox" name="SpeakerComboBox"/>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="HopSizeSpinBox">
            <property name="minimum">
             <number>160</number>
            </property>
            <property name="maximum">
             <number>1024</number>
            </property>
            <property name="value">
             <number>320</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="NumSpeakerSpinBox">
            <property name="minimum">
             <number>2</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="HopSizeLabel">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true">Hop:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QPushButton" name="SaveF0ToNpy">
            <property name="text">
             <string>SaveF0ToNpy</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="NumSpeakerLabel">
            <property name="text">
             <string notr="true">NS:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="SvcEditorSamplingRateBox">
            <property name="minimum">
             <number>8000</number>
            </property>
            <property name="maximum">
             <number>192000</number>
            </property>
            <property name="value">
             <number>44100</number>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QPushButton" name="SaveVolumeToNpy">
            <property name="text">
             <string>SaveVolumeToNpy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="SvcEditorSpeakerTextLabel">
            <property name="text">
             <string>SvcEditorSpeakerTextLabel</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="SvcEditorOpenF0NpyButton">
            <property name="text">
             <string>SvcEditorOpenF0NpyButton</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MCurveEditorPage</class>
   <extends>QWidget</extends>
   <header>editorpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
